toolkit/tools/srpmpacker/srpmpacker.go

```diff
@@ -572,19 +578,20 @@ func specsToPackWorker(requests <-chan string, results chan<- *specState, cancel
 }
 
 // packSRPMs will pack any SPEC files that have been marked as `toPack`.
-func packSRPMs(specStates []*specState, distTag, buildDir string, templateSrcConfig sourceRetrievalConfiguration, workers int) (err error) {
+func packSRPMs(specStates []*specState, distTag, buildDir string, templateSrcConfig sourceRetrievalConfiguration, workers, concurrentNetOps int) (err error) {
 	tsRoot, _ := timestamp.StartEvent("packing SRPMs", nil)
 	defer timestamp.StopEvent(nil)
 	var wg sync.WaitGroup
 
 	allSpecStates := make(chan *specState, len(specStates))
 	results := make(chan *packResult, len(specStates))
 	cancel := make(chan struct{})
+	netOpsSemaphor := make(chan struct{}, concurrentNetOps)
```

Comment:
Nit-pick: why `chan struct{}` instead of `chan int` or other simple and short type? It would make the code a bit shorter.

Reply:
The examples i've seen online, and in our existing code, generally use an empty struct to indicate there is no expected data to pass via the channel.

Reply:
Also `struct{}` has a size of 0. So, in theory, the compiler could optimize around that. (Though I have no idea if it actually does.)